src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/BuildOutput.cs

```diff
@@ -1,12 +1,15 @@
 ﻿using System;
 using System.IO;
 using System.Linq;
+using System.Text.RegularExpressions;
 using Xamarin.Tools.Zip;
 
 namespace Xamarin.ProjectTools
 {
 	public class BuildOutput
 	{
+		Regex regex = new Regex (@"\s+(?<time>\d+)\s+(ms)\s+(" + targetOrTask + @")\s+(?<calls>\d+)\scalls", RegexOptions.Compiled | RegexOptions.ExplicitCapture);
```

Comment:
`targetOrTask` here is a compiler error.

This `GetTargetOrTaskTime` seems useful, though!

Reply:
Yup I moved this RegEx back to the `GetTargetOrTaskTime` method.